### PR TITLE
stacki3: init at 1.0.0

### PR DESCRIPTION
**Pull request by moni-dz**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fortuneteller2k** [comment]
Result of `nixpkgs-review pr 161519` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>stacki3</li>
  </ul>
</details>

File: pkgs/misc/stacki3/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{ lib, poetry2nix, fetchFromGitHub }:
+
+poetry2nix.mkPoetryApplication rec {
+  pname = "stacki3";
+  version = "1.0.0";
+
+  projectDir = fetchFromGitHub {
+    owner = "ViliamV";
+    repo = pname;
+    rev = "8b835f7caed5f10b7cf18f4ee50d7b9695f548e8";
```
Comment (legendofmiracles) [inline pkgs/misc/stacki3/default.nix:10]: Can you add a comment explaining why we're not using the tag?

File: pkgs/misc/stacki3/default.nix [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    hash = "sha256-MzU5YOhL2PvBFAZVfXYQbXAZ/PhOUQg1lX/qnPo/mqY=";
+  };
+
+  pyproject  = ./pyproject.toml;
```
Comment (legendofmiracles) [inline pkgs/misc/stacki3/default.nix:14]: I know nothing about pyproject, but couldn't this be done by referencing the toml file that's in the src?

File: pkgs/misc/stacki3/default.nix [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  poetrylock = ./poetry.lock;
+
+  meta = with lib; {
+    description = "Stack layout for i3/sway wm";
```
Comment (legendofmiracles) [inline pkgs/misc/stacki3/default.nix:18]: Missing homepage

File: pkgs/misc/stacki3/default.nix (hunk 1 above)
Comment (legendofmiracles) [reply pkgs/misc/stacki3/default.nix:11]: Can't we fetch from pypi then?

Comment (fortuneteller2k) [reply pkgs/misc/stacki3/default.nix:11]: Sure, but I will have to introduce `python3Packages` for that.

**Pull request closed without merging**